src/content-tags-drawer/ContentTagsCollapsible.jsx

```diff
@@ -158,11 +174,18 @@ const ContentTagsCollapsible = ({ contentId, taxonomyAndTagsData, editable }) =>
               onChange={handleSelectableBoxChange}
               value={checkedTags}
             >
+              <SearchField
+                onSubmit={() => {}}
```

Comment:
It has no effect when its there because the onChange gets called to perform the search, and the term would already be set. So I opted to set it as a no-op to avoid any surprises.